webui/ocw/views.py

```diff
@@ -54,4 +59,6 @@ def delete(request, key_id=None):
         raise NotImplementedError(
```

Comment:
I can't comment on lines above but you can update lines where you comparing ProviderChoice with your new changes 
if (o.provider == ProviderChoice.AZURE):